espflash/tests/scripts/save-image_write-bin.sh

```diff
@@ -12,8 +12,12 @@ if [[ ! $result =~ "Image successfully saved!" ]]; then
     exit 1
 fi
 echo "Writing binary"
-result=$(espflash write-bin 0x0 app.bin 2>&1)
+result=$(timeout 60 espflash write-bin --monitor 0x0 app.bin --non-interactive 2>&1)
```

Comment:
We already set the timeout in `hil.yml`, is there any reason to have this timeout?